KarhooUISDK/Screens/QuoteList/QuoteListFilters/FiltersViews/NumericFilterView.swift

```diff
@@ -92,15 +93,10 @@ class NumericFilterView: UIView, FilterView {
     }
     
     private func setupLayout() {
-        heightAnchor.constraint(equalToConstant: 52).do {
-            $0.priority = .defaultLow
-            $0.isActive = true
-        }
+        setDimensions(height: 52, priority: .defaultLow)
```

Comment:
Are we able to use some predeffined values instead of hardcoded Int? We have Button.large = 55, mayby it will work?

Reply:
52 is the value from the UI design. In theory, we could change it using 55, but I am afraid the design won't be 'nice' anymore once we'll add additional modifications like that. Our main goal is to create nice UI/UX, all other matters are secondary or less important.

Reply:
Could you bump it locally to 55 and send a screenshot please?

Reply:
After comparing the screenshots, the team agreed to go for 55.